Src/GoogleApis.Auth.DotNet4/OAuth2/GoogleWebAuthorizationBroker.cs

```diff
@@ -113,17 +113,22 @@ public class GoogleWebAuthorizationBroker
         /// <param name="taskCancellationToken">Cancellation token to cancel an operation.</param>
         /// <param name="dataStore">The data store, if not specified a file data store will be used.</param>
         /// <returns>User credential.</returns>
-        private static async Task<UserCredential> AuthorizeAsyncCore(
+        public static async Task<UserCredential> AuthorizeAsyncCore(

```

Comment:
Yeah. It's not a common case, that's why I'm ok with not exposing it anywhere besides this method.
By the way, we should rename it to AuthorizeAsync (no core), not that it's public.